nixos/modules/services/networking/mptcp/mptcp_up_raw

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+# A script for setting up routing tables for MPTCP
+
+# can check with (ins)[root@client:~]# journalctl -b -u NetworkManager-dispatcher.service
+# Copy this script into /etc/network/if-up.d/
+
+set -ex
+
+STATUS=$2
+
+# TODO remove
+echo $PATH
+env > /tmp/if_up_env
+
+RT_TABLE=/etc/iproute2/rt_tables
+
+if [ "$DEVICE_IFACE" = "lo" ]; then
+
+	logger -p user.debug -t mptcp "if localhost or $MODE then abort "
+	exit 0
+fi
+
+if [ -z "$DEVICE_IFACE" ]; then
+	logger -p user.warn -t mptcp "invalid $DEVICE_IFACE"
```

Comment:
Is logging to stdout/stderr not better so it would end up in the journald log of network manager?

Reply:
logger also ends up in journald. I used it because it was in the base example and seemed good enough: like you can set debug levels and a keyword to grep to ("mptcp" here). Now that I am looking to upstream, stdout/stderr may be better.